src/golang/lib/engine/authenticate.go

```diff
@@ -8,8 +8,12 @@ import (
 	"github.com/aqueducthq/aqueduct/lib/lib_utils"
 	"github.com/aqueducthq/aqueduct/lib/workflow/operator/connector/auth"
 	"github.com/dropbox/godropbox/errors"
+	"golang.org/x/sync/errgroup"
 )
 
+// The maximum numnber of concurrent download allowed by Docker and is default to 3.
```

Comment:
nit: "and defaults to 3". Actually I think we can just get rid of the default comment since this is a constant.